timepiece/contracts/models.py

```diff
@@ -60,6 +60,30 @@ def get_admin_url(self):
     def get_absolute_url(self):
         return reverse('view_contract', args=[self.pk])
 
+    def get_noncontract_entries(self, entries=None):
```

Comment:
having a default value of `None` will cause this method to to throw an `AtrtributeError` if no entries are passed in:

```
----> 1 pc.get_noncontract_entries()

...models.py in get_noncontract_entries(self, entries)
     71
     72         for contract in contracts:
---> 73             entries.exclude(
     74                 project__in=contract.projects.all(),
     75                 start_time__gte=contract.start_date,

AttributeError: 'NoneType' object has no attribute 'exclude'

```